onnxruntime/python/tools/quantization/fp16_converter.py

```diff
@@ -0,0 +1,359 @@
+import argparse
+import itertools
+
+import numpy as np
+import onnx
+import packaging.version as pv
+from onnx import helper, numpy_helper
+from onnx import onnx_pb as onnx_proto
+
+from onnxruntime.quantization.onnx_model import ONNXModel
+
+# from onnxruntime.transformers.float16 import convert_tensor_float_to_float16
+
+ALLOWED_OPS_LIST = ["Conv", "MatMul"]
+
+
+def make_value_info_from_tensor(tensor):
+    shape = numpy_helper.to_array(tensor).shape
+    return helper.make_tensor_value_info(tensor.name, tensor.data_type, shape)
+
+
+def _npfloat16_to_int(np_list):
```

Comment:
use type hint?

Reply:
Copied from float16.py

Reply:
I don't see why that would be the reason not to use typehints.